src/index.html

```diff
@@ -1,23 +1,52 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
 	<meta charset="utf-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<meta name="description" content="Update your description here" />
-	<title>Update your title here</title>
+	<title>Mix&Match</title>
 	<!--
 		Webpack will automatically link your CSS file for you.
 		Work in the src/css/styles.scss file.
 	-->
 </head>
+
 <body>
-	<main class="content-container">
-		<h1 class="page-heading">Welcome to your final project!</h1>
-	</main>
-	<!--
-		Webpack will automatically link your JS file for you.
-		Work in the src/js folder.
-	-->
+	<section>
+		<h1 class="logo">Mix&Match</h1>
+		<img class="search" src="images/magnifier.png" alt="search">
+		<img class="menu" src="images/menu.png" alt="menu">
+		<img class="mainpic" src="images/drink.jpg" alt="mainpic">
+		<h2 class="drinkname">Mojito</h2>
+		<p class="main_content">Mojito is a traditional Cuban highball. The cocktail often
+			consists of five ingredients: white rum, sugar, lime juice,
+			soda water, and mint.</p>
+		<div class="container">
+			<h3 class="alcohol_name">RUM</h3>
+			<h3 class="flavor">Fruity</h3>
+			<h3 class="alcohol_level">%LOW</h3>
+		</div>
+	</section>
+	<section>
+		<h4 class="ingre">Ingredients</h4>
+		<ul class="list-ingre">
+			<li><img src="images/alcoh.png" alt="ingre1"></li>
+			<li><img src="images/alcoh.png" alt="ingre2"></li>
+			<li><img src="images/alcoh.png" alt="ingre3"></li>
+			<li><img src="images/alcoh.png" alt="ingre4"></li>
+			<li><img src="images/alcoh.png" alt="ingre5"></li>
```

Comment:
Looks like these images may be in progress. Don't forget to add appropriate alt text when you get the final versions in - in other words, something that will make sense if a screen reader reads it off to a visually impaired user :)